### PR TITLE
Add Makefile error to FilesCheck

### PR DESCRIPTION
**Pull request by scarabeusiv**
This is the only really relevant check from CheckCommonFiles.py
that make sense to be ported.

We check for Makefile(am/in/) in the installed folder and warn if it is present.
We should probably check for CMakelists.txt in the same manner.

Needs a testcase prior merging.

### CODE REVIEW

File: rpmlint/checks/FilesCheck.py [hunk 1]
```diff
@@ -554,6 +555,8 @@ def check(self, pkg):
                 self.output.add_info('W', pkg, 'manifest-in-perl-module', f)
             elif f == '/usr/info/dir' or f == '/usr/share/info/dir':
                 self.output.add_info('E', pkg, 'info-dir-file', f)
+            elif makefile_regex.search(f):
+                self.output.add_info('E', pkg, 'makefile-junk', f)
```
Comment (Conan-Kudo) [inline rpmlint/checks/FilesCheck.py]: I'm not sure this makes sense as an error. For example, the SELinux policy package ships Makefiles as `-devel` content, and there are Makefile-based build systems and tools (blech). I'd downgrade this to a warning.

Comment (scarabeusiv) [reply rpmlint/checks/FilesCheck.py]: Does the selinux put it in some determined location? If by design we could whitelist it and error out on everything else it is probably better.

Comment (Conan-Kudo) [reply rpmlint/checks/FilesCheck.py]: There are various Makefiles in `/usr/share/selinux/devel` and subdirectories.

Comment (scarabeusiv) [reply rpmlint/checks/FilesCheck.py]: So just excluding /usr/share/selinux would be golden right?

Comment (Conan-Kudo) [reply rpmlint/checks/FilesCheck.py]: Probably.

Comment (scarabeusiv) [reply rpmlint/checks/FilesCheck.py]: Added there, now just the tests.

**Pull request merged**